test.config.js

```diff
@@ -41,11 +41,6 @@ try {
   window.scrollTo = window.scrollTo || function() {}
 } catch (error) {}
 
-// Used for Acceptance tests
-require("raf/polyfill")
```

Comment:
can this be removed from package.json?

Reply:
Tests seem to be running fine without it, and I'm assuming based on comments that it was included specifically for acceptance tests.

Reply:
Misread your comment - it is not in package json, seems to be a sub-dependency